### PR TITLE
bhyve: enhance debug info for memory range clash

### PR DESCRIPTION
**Pull request by RobCrowston**
Explain what the two clashing regions are.

### CODE REVIEW

**Comment by jake-87** [comment]
Are we sure that that "claimed" is the right wording? I would lean more towards "reserved", as that seems to be a more accurate description of the message.

File: usr.sbin/bhyve/mem.c [hunk 1]
```diff
@@ -109,9 +109,11 @@ mmio_rb_add(struct mmio_rb_tree *rbt, struct mmio_rb_range *new)
 
 	if (overlap != NULL) {
 #ifdef RB_DEBUG
-		printf("overlap detected: new %lx:%lx, tree %lx:%lx\n",
+		printf("overlap detected: new %lx:%lx, tree %lx:%lx, '%s' "
+		       "claims region already claimed for '%s'\n",
```
Comment (jake-87) [inline usr.sbin/bhyve/mem.c:113]: Reword to "claims region already reserved for"

**Review by jake-87 — CHANGES_REQUESTED** [review]
Reword section for clariry

**Comment by bsdimp** [comment]
This was approved in https://reviews.freebsd.org/D29696 and I landed as efec757b2042

**Pull request closed without merging**